### PR TITLE
[MP-6062] Products Search Widget (Quick Order, Quick add to Cart, Shopping List)

### PR DESCRIPTION
**Pull request by VadymSachenko**
## PR Description

**Documentation ticket**: https://spryker.atlassian.net/browse/MP-6062
**Epic**: https://spryker.atlassian.net/browse/MP-5731

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/user/features/202108.0/marketplace-product-feature-overview.md [hunk 1]
```diff
@@ -37,6 +37,24 @@ The marketplace product is also displayed with the **Sold By** field defining th
 
 ![marketplace product on PDP](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Features/Marketplace+product/add-merchant-product-to-wl-and-from-wh-to-cart.gif)
```
Comment (AdvaBD) [inline docs/marketplace/user/features/202108.0/marketplace-product-feature-overview.md:38]: the gif should be updated to b2b marketplace demo shop

**Review by AdvaBD — CHANGES_REQUESTED** [review]

File: docs/marketplace/user/features/202108.0/marketplace-product-feature-overview.md (hunk 1 above)
Comment (VadymSachenko) [reply docs/marketplace/user/features/202108.0/marketplace-product-feature-overview.md:38]: @ Marketplace Wishlist is a b2b feature, so we can't update it to b2b

Comment (AdvaBD) [reply docs/marketplace/user/features/202108.0/marketplace-product-feature-overview.md:38]: this is a shopping list document not wishlist. and the gif is taking from an outdated demo shop

Comment (VadymSachenko) [reply docs/marketplace/user/features/202108.0/marketplace-product-feature-overview.md:38]: @AdvaBD ok, so I'll remove the outdated git, add a gif for a shopping list, right?

Comment (AdvaBD) [reply docs/marketplace/user/features/202108.0/marketplace-product-feature-overview.md:38]: the shopping list here is only in the context of quick order, it needs to show quick order in the different yves pages

Comment (VadymSachenko) [reply docs/marketplace/user/features/202108.0/marketplace-product-feature-overview.md:38]: Got you, so the give must show Quick Order from all pages:
- Quick Order page
- Quick Add to Cart (on the Cart page)
- Quick Add to Shopping List (on Shopping List View and Edit pages)

Comment (AdvaBD) [reply docs/marketplace/user/features/202108.0/marketplace-product-feature-overview.md:38]: yes :-)

**Review by gibmirmilch — APPROVED** [review]

**Pull request merged**